### ISSUE TITLE
Impossible Alien Artifact Rooms

### ISSUE THREAD

**Issue by Kaillera** [Regalis11/Barotrauma#2211]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
[Artifact Rooms with Trap] that generates smaller than normal are made impossible to enter due to Alien Coil range or the Motion Sensor range extending to the doors.

![Capture](https://user-images.githubusercontent.com/5086682/74072934-916ead00-49d6-11ea-9a4c-a34548fb815f.PNG)

It also doesn't help that EMP grenades doesn't work on them.

**Version**
~0.9.704

**Comment by Udrakan** [comment]
I've encountered these ruins before. I've used steroids on myself and inched forward everytime the EMP gave me a pause. This room seems kinda unfair to be honest. Unless theres something we are missing.

**Comment by Kaillera** [comment]
> I've encountered these ruins before. I've used steroids on myself and inched forward everytime the EMP gave me a pause. This room seems kinda unfair to be honest. Unless theres something we are missing.

It's not your fault. Most of them, when generated properly will let you slip past with minor shock damage.

**Comment by NilanthAnimosus** [comment]
The only time I ever saw this room, I always tried the sides/to slip past but no matter where I tried to hide or swim it was always a guaranteed death (I even tried spamming healing through one). After my third/fourth try I simply on sight learned to never bother with the artifact if I saw this generate.

Never thought to think of an absolute death trap as a potential bug, That looks normal size for what I remember oddly enough.
